ls1intum/Artemis
# `Exam mode`: Enable the creation of test exams for instructors

### Checklist
#### General
- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.(https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
#### Client

### Motivation and Context
The creation of Test Exams is currently only available for admins, as the underlying structure to conduct and manage a Test Exam was missing.

### Description
With Exam mode: Conduct test exams #5194  and Exam mode: Improve management for test exams #5204 , the structure is now added and thus the creation of test exams for Instructors can be enabled.
I.e. the check for admin rights when creating a new Exam is removed.

### Steps for Testing
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration -> Exams
3. Create a new exam and make sure, that you can select between the two different modes "Exam" and "Test Exam".

#### Code Review
#### Manual Tests

### Screenshots
![image](https://user-images.githubusercontent.com/94070506/179973196-923e5819-e717-41ea-ab94-16bd0ad280ff.png)
